tools/ci_build/github/linux/docker/Dockerfile_manylinux2014_openvino_multipython

```diff
@@ -0,0 +1,82 @@
+FROM quay.io/pypa/manylinux2014_x86_64:latest
```

Comment:
So what is this file for? It looks like it is very different than the other docker files in this dir  because it takes dependency on ORT code, and build arbitrary branches, not the current code.  It makes feel that you put this file in wrong place.

Reply:
This file is to build OpenVINO Execution Provider for ORT. Last time we needed to build a special branch but going forward we can build on master. Anyways the input branch is a Docker argument so by default we can keep it as master

Reply:
But the other docker files in this directory do not do "git checkout https://github.com/Microsoft/onnxruntime". Why does this one need to? 
And, for you information, I'm asked to remove all references to "quay.io/pypa/***:***". For now I will not touch this file. But I want to know more about this file, how is it used and how to test it when I need to make changes to it?